### PR TITLE
fix: Improve copy+paste readiness of release guide

### PR DESCRIPTION
**Pull request by kriskowal**

### CODE REVIEW

File: MAINTAINERS.md [hunk 1]
```diff
@@ -91,11 +91,11 @@
   `npm login`. You can check with `npm whoami`.
 
   ```
```
Comment (mhofman) [inline MAINTAINERS.md:93]:
```suggestion
  ```sh
```

Maybe we could use the shell syntax highlight?

File: MAINTAINERS.md [hunk 2]
```diff
@@ -106,7 +106,7 @@
   a distressingly long time. Instead you can verify the version with
 
   ```
```
Comment (mhofman) [inline MAINTAINERS.md:108]:
```suggestion
  ```sh
```

Same here

**Review by mhofman — APPROVED** [review]

**Comment by mhofman** [comment]
Btw, any reason this uses `npm` to publish and not `yarn`?

**Comment by kriskowal** [comment]
We are using yarn to publish. We’re using npm for credentials. I don’t know why it works. `yarn login` presumably shells out to `npm login` and `yarn whoami` doesn’t exist. 🤷‍♂️

**Pull request merged**